config.yaml

```diff
@@ -367,59 +367,127 @@ languages:
-          url: "/tracing/languages/java/"
-          weight: 55
-          parent: customnav_tracingnav_langs
+          url: "/tracing/setup/java/"
+          weight: 140
+          parent: customnav_tracingnav_setup
         - identifier: customnav_tracingnav_python
           name: "Python"
-          url: "/tracing/languages/python/"
-          weight: 60
-          parent: customnav_tracingnav_langs
+          url: "/tracing/setup/python/"
+          weight: 150
+          parent: customnav_tracingnav_setup
         - identifier: customnav_tracingnav_ruby
           name: "Ruby"
-          url: "/tracing/languages/ruby/"
-          weight: 70
-          parent: customnav_tracingnav_langs
-        - identifier: customnav_tracingnav_docker
-          name: "Docker"
-          url: "/tracing/docker/"
-          weight: 80
+          url: "/tracing/setup/ruby/"
+          weight: 160
+          parent: customnav_tracingnav_setup
+        - identifier: customnav_tracingnav_librairies
+          name: "Community Librairies"
+          url: "/libraries/#apm-tracing-client-libraries"
+          weight: 170
+          parent: customnav_tracingnav_setup
+
+        # Services
+        - identifier: customnav_tracingnav_services
+          name: "Services List"
+          url: "/tracing/services/"
+          weight: 20
+          parent: menu_references_tracing
+        - identifier: customnav_tracingnav_service
+          name: "Service"
+          url: "/tracing/services/service"
+          weight: 210
+          parent: customnav_tracingnav_services
+        - identifier: customnav_tracingnav_resource
+          name: "Resource"
+          url: "/tracing/services/resource"
+          weight: 220
+          parent: customnav_tracingnav_services
+        - identifier: customnav_tracingnav_trace
+          name: "Trace"
+          url: "/tracing/services/trace"
+          weight: 230
+          parent: customnav_tracingnav_services
+
+        # Traces
+        - identifier: customnav_tracingnav_traces
+          name: "Traces Search"
+          url: "/tracing/traces/"
+          weight: 30
           parent: menu_references_tracing
+
+        # Trace API
         - identifier: customnav_tracingnav_api
           name: "Trace API"
           url: "/api/#tracing"
-          weight: 90
+          weight: 410
           parent: menu_references_tracing
+
+        # Trace monitor
+        - identifier: customnav_tracingnav_apm_monitor
+          name: "APM Monitor"
+          url: "/monitors/monitor_types/apm"
+          weight: 420
+          parent: menu_references_tracing
+
+        # Trace Dashboards
+        - identifier: customnav_tracingnav_apm_widget
+          name: "APM in Dashboard"
+          url: "/graphing/dashboards/widgets/#service-summary"
+          weight: 430
+          parent: menu_references_tracing
+
+        # Tracing Product Specs
+        - identifier: customnav_tracingnav_product_specs
+          name: "Product specs"
```

Comment:
We're not consistent with capitalization on the side bar.

Reply:
👌